training/tests.yml

```diff
@@ -6,3 +6,11 @@ vision:
       job: "./vision/jobs/object-detection-using-built-in-component/pipeline.yml"
       pre: "./vision/jobs/object-detection-using-built-in-component/prepare_data.py"
 
+automl_components:
+  includes:
+  - ./automl/components/
```

Comment:
the paths under includes should be paths for tests/jobs, not assets. Assets will be automatically converted to the release branch.
For your case, you probably want to include `./automl/tests` and the workflow will copy the `tests` to the release branch for later testing.

Reply:
Updated thanks